apko/private/apko_image.bzl

```diff
@@ -58,17 +95,25 @@ def _impl(ctx):
     for content in depset(transitive = deps).to_list():
         content_owner = content.owner.workspace_name
         content_cache_entry_key = content.path[content.path.find(content_owner) + len(content_owner) + 1:]
-        content_entry = ctx.actions.declare_file("/".join([cache_name, content_cache_entry_key]))
+        content_entry = ctx.actions.declare_file(paths.join(workdir, cache_name, content_cache_entry_key))
         ctx.actions.symlink(
             target_file = content,
             output = content_entry,
         )
         inputs.append(content_entry)
 
-    ctx.actions.run(
-        executable = apko_info.binary,
+    apko_binary = ctx.actions.declare_file(paths.join(workdir, apko_info.binary.short_path))
+    ctx.actions.symlink(
+        target_file = apko_info.binary,
+        output = apko_binary,
+    )
+    inputs.append(apko_binary)
+
+    ctx.actions.run_shell(
+        command = "cd {}/{} && {} $@".format(paths.join(ctx.bin_dir.path, ctx.label.workspace_root, ctx.label.package), workdir, apko_info.binary.short_path),
```

Comment:
can we add `workdir` to the join, to avoid `{}/{}`

Reply:
Done